add adguard for windows v8 section (#785)
AdguardTeam/KnowledgeBase: docs/adguard-for-windows-8/protection/annoyance-blocking.md

---
title: Annoyance blocking
sidebar_position: 3
---

:::info

This article describes the beta version of AdGuard for Windows v8.0, a comprehensive ad blocker that protects your device at the system level.

> This is a beta release and still under development. If you’d like to try it, download the [beta version of AdGuard for Windows v8](https://agrd.io/adguard_for_windows_8_beta).

Reviewer: same comment as above

:::

> Annoyance filters are designed to improve the usability of websites by blocking elements on pages that are not advertisements but distract from and interfere with site interaction or content consumption — such as various modal windows and interactive forms, cookie consent notifications and requests, mobile app banners, and various widgets.

Reviewer: ```suggestion
Annoyance filters are designed to improve website usability by blocking non-advertisement elements that distract from or interfere with site interaction or content consumption. These elements include modal windows, interactive forms, cookie consent notifications and requests, mobile app banners, and various widgets.
```

> This filter may contain rules that are not suitable for all users. Sometimes it is recommended to disable this filter. The decisions to add rules to this filter are made by filter developers on a rule-by-rule basis.

Reviewer: These filters?
